lib/exercises.rb

```diff
@@ -1,20 +1,67 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n log n)
+# Space Complexity: O(n)
 
 def grouped_anagrams(strings)
```

Comment:
Since you're sorting the words and English words are small this method actually depends on the number of words, not the length of them.  

So it's O(n * m log m) where m is the length of the words, or if you assume the words are short O(n)